api/preprints/serializers.py

```diff
@@ -76,6 +77,10 @@ class PreprintSerializer(JSONAPISerializer):
     is_preprint_orphan = ser.BooleanField(read_only=True)
     license_record = NodeLicenseSerializer(required=False, source='license')
 
+    reviews_state = ser.CharField(read_only=True, max_length=15)
+
+    submit_for_review = ser.BooleanField(required=False, write_only=True)
```

Comment:
(`submit_for_review` is only needed for editing/resubmitting. For creation, there's already logic to recognize when `is_published` is set for the first time, which triggers the `submit` transition)